cypress/integration/specialFeatures/scriptSwitching/testsForCanonicalOnly.js

```diff
@@ -0,0 +1,75 @@
+import visitPage from '../../../support/helpers/visitPage';
+import {
+  allVariantAssertions,
+  nonCookieVariantAssertions,
+} from '../utilities/scriptSwitchingJourneyAssertions';
+import {
+  clickScriptSwitcher,
+  clickHomePageLink,
+  clickPromoLinkOnHomePage,
+} from '../utilities/scriptSwitchingJourneyActions';
+import {
+  getPrivacyBannerAccept,
+  getCookieBannerAccept,
+} from '../utilities/cookiePrivacyBanner';
+
+export default ({
+  serviceId,
+  serviceName,
+  pageType,
+  path,
+  variant,
+  otherVariant,
+}) => {
+  describe(`Script Switching - ${serviceId} - ${pageType} - ${path}`, () => {
+    beforeEach(() => {
+      cy.clearCookies();
+      visitPage(path, pageType);
+    });
+
+    it(`should change to the correct script when switching script between ${variant} and ${otherVariant}`, () => {
+      // Accept privacy banner
+      getPrivacyBannerAccept(serviceId, variant).click();
+
+      // Accept cookie banner
+      getCookieBannerAccept(serviceId, variant).click();
+
+      cy.log(
+        `Asserting script switch button, url and document lang against variant: ${variant}`,
+      );
+      nonCookieVariantAssertions(serviceName, variant);
+
+      // Clicks script switcher
+      clickScriptSwitcher(otherVariant);
+
+      cy.log(
+        `Asserting cookies, script switch button, url and document lang against other variant: ${otherVariant}`,
+      );
+      allVariantAssertions(serviceName, otherVariant);
+
+      // Navigate to home page by clicking link in the banner
+      clickHomePageLink(serviceName);
+
+      cy.log(
+        `Asserting cookies, script switch button, url and document lang has persisted for other variant: ${otherVariant}`,
```

Comment:
As per previous comments - if we're not checking cookies, let's remove all references. Thanks.